sklearn/decomposition/tests/test_nmf.py

```diff
@@ -42,25 +42,28 @@ def test_initialize_nn_output():
 def test_parameter_checking():
     A = np.ones((2, 2))
     name = 'spam'
+    # FIXME : should be removed in 0.26
+    init = 'nndsvd'
```

Comment:
I think we should make sure now that the tests pass with the future default `init = 'nndsvda'`.
```suggestion
    init = 'nndsvda'
```
(in other tests too)